src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Publish.targets

```diff
@@ -192,6 +192,12 @@ Copyright (c) .NET Foundation. All rights reserved.
         <RelativePath>@(_DebugSymbolsIntermediatePath->'%(Filename)%(Extension)')</RelativePath>
       </ResolvedFileToPublish>
 
+      <!-- Copy the documentation file (.xml), if any -->
+      <ResolvedFileToPublish Include="@(_DebugSymbolsIntermediatePath)"
```

Comment:
Thanks, Daniel. I'll give this a try.

In trying to repro the issue in a test Friday, I noticed that TestLibrary.csproj under KitchenSink isn't set to create documentation files. I thought I would have a repro by adding them but no luck. I added the documentation files to the build output and they did get published, causing the test I was looking at (`GivenThatWeWantToPublishAProjectWithAllFeatures.It_publishes_the_project_correctly`) to fail because the doc files were not in the expected output list. So SDK's tests seem to be doing something different from the CLI and don't exhibit the bug. However, the change above, when used in the CLI, does fix the bug. 

This is my first foray into the SDK codebase so any insights are welcome. I definitely plan to get a test around this.